### ISSUE TITLE
Discussion template does not seem to work

### ISSUE THREAD

**Issue by rugk** [PrivateBin/PrivateBin#1143]
The issue templates introdued in #1134 seem to work. But not the qa discussion template. The discussion is open as always.

<-- Describe the bug: A clear and concise description of what the bug is. -->
^^ and btw wtf is that why does it keep showing that comment

Btw we may also create/choose a new discussion for that but well anyway.

Doc https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/configuring-issue-templates-for-your-repository#creating-issue-forms

**Comment by r4sas** [comment]
> <-- Describe the bug: A clear and concise description of what the bug is. -->
> ^^ and btw wtf is that why does it keep showing that comment

Because it must be `<!--`

**Comment by elrido** [comment]
I tried finding other repos with examples and the slug is correct. Then I noticed this: If you go to https://github.com/PrivateBin/PrivateBin/blob/master/.github/DISCUSSION_TEMPLATE/q-a.yml it does show a validation error, which I will now try to address in a PR (incl. @r4sas finding).

When looking at other projects templates, there is even a preview of the form:
- https://github.com/aquasecurity/trivy/blob/main/.github/DISCUSSION_TEMPLATE/q-a.yml
- https://github.com/codemagic-ci-cd/codemagic-docs/blob/master/.github/DISCUSSION_TEMPLATE/q-a.yml

For future reference, the validation error displayed was:
> There is a problem with this template
> 
> body[11]: radio is not a valid input type. [Learn more about this error.](https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/common-validation-errors-when-creating-issue-forms#bodyi-x-is-not-a-valid-input-type)

**Comment by elrido** [comment]
*Sigh* The updated [Q&A template still has issues](https://github.com/PrivateBin/PrivateBin/blob/master/.github/DISCUSSION_TEMPLATE/q-a.yml). It now reports:

```
body[0]: Required attribute key value is missing. Learn more about error 1.
body[0]: label is not a permitted attribute. Learn more about error 2.
body[0]: description is not a permitted attribute. Learn more about error 3.
body[2]: Required attribute key value is missing. Learn more about error 4.
body[2]: label is not a permitted attribute. Learn more about error 5.
body[2]: description is not a permitted attribute. Learn more about error 6.
```

**Comment by r4sas** [comment]
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-issue-forms
https://docs.github.com/en/communities/using-templates-to-encourage-useful-issues-and-pull-requests/syntax-for-githubs-form-schema

**Comment by elrido** [comment]
FYI, it needed one more tweak, quoting the booleans, and now it finally works: https://github.com/orgs/PrivateBin/discussions/new?category=q-a

**Comment by r4sas** [comment]
A bit more fixes to labels and descriptions in #1150.
